src/games/yinsh.cc

```diff
@@ -22,7 +22,7 @@
 // draw ka rule is different- no of rings pe aa jaati hai baat
 // namespace Yinsh{
 
-uint64_t StateForYinsh::hash_table[5][BOARD_X][BOARD_Y] = {0};
```

Comment:
how was this ever working?!

Reply:
It's legal to specify an array initializer using a single set of brackets, but its' clearer to do all 3 levels.